vhdbuilder/packer/install-dependencies.sh

```diff
@@ -87,6 +87,10 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
   containerd_version="1.4.4"
   installStandaloneContainerd ${containerd_version}
   echo "  - containerd v${containerd_version}" >> ${VHD_LOGS_FILEPATH}
+  # also pre-cache containerd 1.5 for ACC as a local .deb file
+  containerd_version="1.5.0-beta.git31a0f92df"
+  downloadContainerd ${containerd_version}
+  echo "  - containerd v${containerd_version}" >> ${VHD_LOGS_FILEPATH}
```

Comment:
nit: can we make it explicit that it's just a cache of package? otherwise we cannot tell which one is effective

Reply:
yea good call, will change